tests/accounts.test.js

```diff
@@ -0,0 +1,760 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const Accounts = require('../src/models/accounts');
+const mongoose = require('mongoose');
+const {server} = require('../src/app');
+chai.should();
+chai.use(chaiHttp);
+let accessToken= null;
+
+describe("GET Tests Accounts Filters",()=>{
+    before((done) =>{
+        const login = {
+            username: "Karla",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    describe("GET Tests Accounts Lastname", () => {
+        it("GET /Accounts Lastname", (done) =>{
+            chai.request(server).get("/accounts/lastname/Ortiz")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(200);
+                response.body.should.be.a("array");
+                response.body.length.should.be.eq(1);
+                done();
+            });
+        });
+
+        it("GET /Accounts Lastname Not Found", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        it("GET /Accounts Lastname Not Found", (done) => {
```